src/redux/greeting.js

```diff
@@ -0,0 +1,21 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const apiUrl = 'http://localhost:3000/api/greetings';
+/* eslint-disable */
+export const fetchGreeting = createAsyncThunk('greeting/fetchGreeting', () =>
+  fetch(apiUrl)
+    .then((response) => response.json())
+    .then((json) => json),
+);
+
+/* eslint-enable */
```

Comment:
- [ ] **[OPTIONAL]** You did a great job working on this Milestone! 🚀  However, please avoid disabling linters as they help ensure a consistent code format as well as apply best practices. I'm marking this as optional because I can't see any error when I deleted it, so please consider removing it! 🤓 🔧